### ISSUE TITLE
Claw toe deformity

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7769]
**Preferred term label:**
Claw toe deformity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Claw toes are characterized by hyperextension at the metatarsal-phalangeal joints and flexion of the interphalangeal joints
Pes cavus and claw toes are different from pes equinus, a more common manifestation of spasticity which precedes those
deformities.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
616282 
PMID: 25751282  2/5

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
